### PR TITLE
[KYUUBI #3040] [Subtask][#3039] Build the skeleton of client side documentation

### PR DESCRIPTION
**Pull request by yaooqinn**
### _Why are the changes needed?_

As a subtask of #3039, this PR aims to build the skeleton of client side documentation

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate
image

image

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: docs/client/jdbc/kyuubi_jdbc.rst [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+       <artifactId>kyuubi-hive-jdbc-shaded</artifactId>
+       <version>1.5.2-incubating</version>
+   </dependency>
+
+Sbt
+^^^
+
+.. code-block:: sbt
+
+   libraryDependencies += "org.apache.kyuubi" % "kyuubi-hive-jdbc-shaded" % "1.5.0-incubating"
```
Comment (pan3793) [inline docs/client/jdbc/kyuubi_jdbc.rst:62]: 1.5.2

File: docs/client/python/pyhive.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+
+`PyHive`_
+=========
+
+.. warning:: The document you are visiting now is incomplete, please help kyuubi community to fix it if appropriate for you.
+
+.. _PyHive: https://github.com/dropbox/PyHive
```
Comment (pan3793) [inline docs/client/python/pyhive.rst:22]: The forked https://github.com/acryldata/PyHive has better support for Kyuubi/Spark

File: docs/client/bi_tools/dbeaver.rst [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+DBeaver
+=======
+
+What is DBeaver
+---------------
+
+.. image:: https://raw.githubusercontent.com/wiki/dbeaver/dbeaver/images/dbeaver-icon-64x64.png
```
Comment (cfmcgrady) [inline docs/client/bi_tools/dbeaver.rst:22]: nit: a local image is a better choice?

**Review by ulysses-you — APPROVED** [review]

**Review by pan3793 — APPROVED** [review]

**Review by cxzl25 — APPROVED** [review]

**Comment by yaooqinn** [comment]
thanks @cfmcgrady @cxzl25 @ulysses-you @pan3793 for the review, merged to master

**Pull request closed without merging**